src/assets/src/components/queueEditors.tsx

```diff
@@ -74,6 +79,27 @@ export function GeneralEditor(props: GeneralEditorProps) {
                 onChange={props.onChangeAllowed}
                 disabled={props.disabled}
             />
+            {/* our code */}
+            {
+                props.allowedMeetingTypes.has("inperson") &&
+                <div>
```

Comment:
You can also use `<>` and `</>` here instaed of an enclosing `div`. I recently saw a tip in an article suggesting this was preferred, see 4 in https://betterprogramming.pub/21-best-practices-for-a-clean-react-project-df788a682fb. Certainly it would keep the HTML structure more consistent, since none of the blocks associated with a field are grouped in `div`s.